### PR TITLE
DSA_1.1.3.4.Arrays_Range_Queries_410

### LINKED ISSUE #410: DSA 1.1.3.4_Arrays: Merge Sort Tree - Audio/Video
## Description
Explain about Merge Sort Tree.

## Task
- [x] Documentation
- [ ] Audio
- [ ] Video

## Note
- While requesting the issue to be assigned, please mention your **NAME & BATCH NUMBER**.
- Kindly refer to [Issue Guidelines](https://github.com/girlscript/winter-of-contributing/blob/DSA/DSA/CONTRIBUTING.md#issues).
- In case you see that the issue has already been assigned, try to request for a different task within the same issue.
- Changes should be made inside the ````DSA/directory```` & ````DSA```` branch.
- Name of the file should be same as the issue.
- This issue is only for 'GWOC' contributors of 'DSA' Domain.

### PR DESCRIPTION
**Pull request by Harsh-Chaudhary7**
<hr>

## Description 📜

I have documented the explanation of Merge sort tree and its implementation too. please review it.@nimisha-yadav 
Fixes #410

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] DSA

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

<hr>

### CODE REVIEW

File: DSA/1.1 Arrays/3. Range Queries/DSA_1.1.3.4_Arrays_ Range_Queries_#410.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+## Problem Statement
+Given an array of N elements and you have to answer Q queries of the form L R K , To Count the numbers
```
Comment (harshsri28) [inline DSA/1.1 Arrays/3. Range Queries/DSA_1.1.3.4_Arrays_ Range_Queries_#410.md:2]: update the problem statement something like explanation of merge sort tree and it's implementation , rest is looking fine.

**Review by harshsri28 — CHANGES_REQUESTED** [review]
update the problem statement something like explanation of merge sort tree and it's implementation , rest is looking fine.

**Comment by Harsh-Chaudhary7** [comment]
> update the problem statement something like explanation of merge sort tree and it's implementation , rest is looking fine.
Updation done please review it. @harshsri28

**Review by harshsri28 — APPROVED** [review]
LGTM

**Comment by Harsh-Chaudhary7** [comment]
Please review it @nimisha-yadav.

**Review by nimisha-yadav — CHANGES_REQUESTED** [review]
LGTM. Just rename your file 1.1.3.4_Merge_Sort_Tree.md

**Comment by Harsh-Chaudhary7** [comment]
Renaming done @nimisha-yadav

**Review by nimisha-yadav — APPROVED** [review]
Congrats on getting your PR merged!

**Pull request merged**